packages/pharos/src/components/link/PharosLink.react.stories.mdx

```diff
@@ -24,6 +24,14 @@ import { PharosLink } from '../../react-components/link/pharos-link';
 
 <ArgsTable of={PharosLink} />
 
+# Button
+
+<Canvas withToolbar>
+  <Story name="Button">
+    <PharosLink>I am a button</PharosLink>
```

Comment:
Could you add the other link variants to validate all of them look correct as buttons?

Reply:
You can have them all in this same story.

Reply:
Added variants to button story, so resolving this conversation.